reco_utils/dataset/wikidata.py

```diff
@@ -55,11 +57,16 @@ def find_wikidata_id(name, limit=1, session=None):
 
     try:
         response = session.get(API_URL_WIKIPEDIA, params=params)
-        page_id = response.json()["query"]["search"][0]["pageid"]
     except Exception as e:
```

Comment:
Isn't `Exception` too broad? Is "badRequest" the only exception we may get from session.get when something went wrong?

Reply:
yeah, you are completely right. Yesterday I was trying to do exactly that, and separate the exceptions, however, I got different responses and even bad requests, so I put the code in its original form. I think we should review this in the future